.github/workflows/smoke-test-release-run-ubuntu.yml

```diff
@@ -0,0 +1,101 @@
+name: "Positron: Run smoke tests against latest release"
```

Comment:
Nit: These long names don't display well in GHA. some ideas....
*  `Latest Release: E2E Electron Tests`
* `Positron (Release): Smoke Tests`

Actually, do we need to call out Positron? Duh, it's in the Positron repo.... so isn't that obvioius?
* `Release: Run Smoke Tests`
* `Latest Release: Smoke Tests` (are we going to add web too?)
* `Latest Release: E2E Electron Tests` (technically, they are only electron tests in the workflow)

You don't have to change it, just a suggestion... :)

Reply:
I will go with Latest Release: E2E Electron Tests